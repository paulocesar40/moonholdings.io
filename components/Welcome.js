import React from 'react';

import { welcome } from '../styles';

const { WelcomeContainer, WelcomeMessage } = welcome;

class Board extends React.Component {
  render() {
    return (
      <WelcomeContainer>
        <WelcomeMessage>
          <h1>MOON.HOLDINGS</h1>
          <h4>A Futuratum Project</h4>
          <h2>Click the <span>+</span> button to create your portfolio.</h2>
        </WelcomeMessage>
      </WelcomeContainer>
    );
  }
}

export default Board;
