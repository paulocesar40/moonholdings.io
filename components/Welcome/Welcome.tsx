import React from 'react'

import { moonHoldings } from '../../shared/models'
import { welcome } from '../../styles'

const { WelcomeContainer, WelcomeMessage } = welcome;

export default class Board extends React.Component {
  render() {
    return (
      <WelcomeContainer>
        <WelcomeMessage>
          <h1>{moonHoldings}</h1>
          <h4>A Futuratum Project</h4>
          <h2>Click the <span>+</span> button to create your portfolio.</h2>
        </WelcomeMessage>
      </WelcomeContainer>
    );
  }
}
