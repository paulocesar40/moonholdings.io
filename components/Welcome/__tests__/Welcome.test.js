import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Welcome from '../Welcome'

describe('<Astronaut /> component', () => {
  describe('rendering', () => {
    const wrapper = shallow(<Welcome />);
    it('should render a component matching the snapshot', () => {
      const tree = toJson(wrapper);
      expect(tree).toMatchSnapshot();
      expect(wrapper).toHaveLength(1);
    });
  });
});
