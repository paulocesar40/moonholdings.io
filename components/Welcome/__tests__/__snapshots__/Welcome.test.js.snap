// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Astronaut /> component rendering should render a component matching the snapshot 1`] = `
<welcome__WelcomeContainer>
  <welcome__WelcomeMessage>
    <h1>
      MOON.HOLDINGS
    </h1>
    <h4>
      A Futuratum Project
    </h4>
    <h2>
      Click the 
      <span>
        +
      </span>
       button to create your portfolio.
    </h2>
  </welcome__WelcomeMessage>
</welcome__WelcomeContainer>
`;
